app/config/collections/platform.php

```diff
@@ -689,6 +700,107 @@
         ],
     ],
 
+    'devKeys' => [
+        '$collection' => ID::custom(Database::METADATA),
+        '$id' => ID::custom('devKeys'),
+        'name' => 'Dev keys',
+        'attributes' => [
+            [
```

Comment:
Let's add search attribute

Reply:
Do not include in response model, but allow to query by it. Ensure it is also set when creating document, and updated when updating. Make it be combination of dev key ID, project ID, and key name.